python/packages/nisar/antenna/rx_channel_imbalance_helpers.py

```diff
@@ -0,0 +1,434 @@
+from __future__ import annotations
+from warnings import warn
+from typing import Tuple, Dict
+from dataclasses import dataclass
+
+import numpy as np
+import h5py
+
+from nisar.products.readers.Raw import Raw
+from nisar.antenna import get_calib_range_line_idx, CalPath
+from isce3.core import speed_of_light
+
+
+@dataclass(frozen=True)
+class RX_CHANNEL_IMBALANCE_PRODUCT:
```

Comment:
```suggestion
class RxChannelImbalanceProduct:
```
All-caps names like these are typically reserved for constants in Python, is there a reason why you named the class in this way?